code_review_guide.md

```diff
@@ -0,0 +1,48 @@
+#Definition:
+
+* Code review is the process in which code written by one or more developers is inspected by another developer to look for defects and improvements.
+
+#Goals of Code Review:
+
+* Simplify Code
+* Make sure acceptance criteria are met
+* Keep code maintanable
+* Maintain code style
+* Keep good test coverage
+* Ensure code is commented, so a new person can understand the logic
```

Comment:
I would change commented to documented because comments although sometimes the correct answer have their drawbacks. Documentation is done firstly by good naming object and methods, specs, High Level ReadMe, Raml Docs, and then comments. Comment should be a last resort after all others have been evaluated.